components/search/themes/spectrum-two.css

```diff
@@ -87,7 +87,6 @@
 	}
 
 	.spectrum-Search--quiet {
-		--spectrum-search-quiet-button-offset: calc((var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2) - (var(--mod-workflow-icon-size-100, var(--spectrum-workflow-icon-size-100)) / 2));
```

Comment:
Is this already moved into the search component's index.css so that we're maintaining this offset calculation?

Reply:
Yeah, I saw it was already being overridden by this existing line in index.css:
https://github.com/adobe/spectrum-css/blob/61868f94cac1a0f9918fa6c60679c2970750ae16/components/search/index.css#L160

I noticed that this is slightly different than main though:
`--spectrum-search-quiet-button-offset: calc((var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2) - (var(--mod-workflow-icon-size-100, var(--spectrum-search-icon-size)) / 2));` 
as compared to 
`--spectrum-search-quiet-button-offset: calc((var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2) - (var(--mod-workflow-icon-size-100, var(--spectrum-workflow-icon-size-100)) / 2));` 

I think that was to remove the global token reference from the index.css? It results in slightly different behavior, because `--spectrum-search-icon-size` changes per t-shirt size (and the mod name should be `--mod-search-icon-size`). Using `--spectrum-search-icon-size` looks like it would be more correct and a fix...but the more I look at the quiet close button positioning, and look at React Spectrum's version, I think the positioning is incorrect either way; I can open a bug issue to look into fixing that in main. For now, should I keep this as is, but rename the mod?

Reply:
I've changed the mod name for now here: [948da81](https://github.com/adobe/spectrum-css/pull/3023/commits/948da81cf75ca01b542147f3de81f3a9cd0ab638)

I've created a Jira issue CSS-914 and started a discussion with design to confirm about the correct alignment of the clear button for quiet. Whether there are changes or not, it looks like this needs a bugfix on `main` first.